app/src/main/java/com/mealsmadeeasy/inject/MealsComponent.kt

```diff
@@ -28,4 +29,6 @@ interface MealsComponent {
     fun inject(fragment: GroceryListFragment)
     fun inject(fragment: DiscoverFragment)
     fun inject(fragment: EditServingsDialogFragment)
+
+    fun inject(activity: GroceryListAdapter)
```

Comment:
I think there's a naming problem here (`GroceryListAdapter` isn't an `Activity`). I'd prefer to inject into the fragment/activity instead, and just pass the `MealStore` as a parameter instead. If you need to keep this inject method, then go ahead and group it with its activity/fragment friends.